src/InteractiveWindow/Editor/Commands/ClearScreenCommand.cs

```diff
@@ -20,12 +17,13 @@ public override Task<ExecutionResult> Execute(IInteractiveWindow window, string
 
         public override string Description
         {
-            get { return "Clears the contents of the REPL editor window, leaving history and execution context intact."; }
+            // TODO: Needs localization...
+            get { return "Clears the contents of the editor window, leaving history and execution context intact."; }
         }
 
         public override IEnumerable<string> Names
         {
-            get { yield return "cls"; }
+            get { yield return "cls"; yield return "clear"; }
```

Comment:
@DinoV @zooba is it problematic if "$clear" is aliased to clear screen in Python?